packages/api/src/main/messaging/MessagingClient.ts

```diff
@@ -0,0 +1,184 @@
+/*
+ * Copyright IBM Corp. 2021, 2021
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import amqp from 'amqplib'
+import { v4 as uuidv4 } from 'uuid'
+
+import { EventMessage, QueryMessage } from './config'
+import { MESSAGE_QUEUE_URL } from './constants'
+
+const RANDOM_QUEUE_NAME = ''
+const DEFAULT_ROUTING_KEY = ''
```

Comment:
are these values meant to be empty?

Reply:
Yeah. I added some docs to these to clarify.